components/style/properties/data.py

```diff
@@ -75,7 +75,8 @@ def __init__(self, name, values, gecko_constant_prefix=None,
         self.extra_gecko_aliases = parse_aliases(extra_gecko_aliases or "")
         self.extra_servo_aliases = parse_aliases(extra_servo_aliases or "")
         self.consts_map = {} if custom_consts is None else custom_consts
-        self.gecko_strip_moz_prefix = gecko_strip_moz_prefix
+        self.gecko_strip_moz_prefix = True \
```

Comment:
Because when `single_keyword_computed` generates `keyword_kwargs`, it assigns `None` to `gecko_strip_moz_prefix` key if it isn't specified, and `None` is equal to false in condition, which reverses the default value of this parameter.